bin/improver-temp-lapse-rate

```diff
@@ -94,14 +102,24 @@ def main():
     orography_cube = load_cube(args.orography_filepath)
     land_sea_mask_cube = load_cube(args.land_sea_mask_filepath)
 
-    result = LapseRate(
-        max_height_diff=args.max_height_diff,
-        nbhood_radius=args.nbhood_radius,
-        max_lapse_rate=args.max_lapse_rate,
-        min_lapse_rate=args.min_lapse_rate).process(temperature_cube,
-                                                    orography_cube,
-                                                    land_sea_mask_cube)
-
+    if args.return_dalr:
+        result = temperature_cube.copy(
```

Comment:
I don't agree in this case.  All we are doing is copying a cube with template data and renaming it.  This is all basic library functionality and extremely simple - analogous to the "merge cubes and add metadata" lines in the weighted blending CLI (you wouldn't factor 2 function calls out into another function just because they're called in sequence).